dockstore-webservice/src/main/resources/migrations.1.12.0.xml

```diff
@@ -102,4 +102,28 @@
             create unique index collection_displayname_index on collection (LOWER(displayname), organizationid);
         </sql>
     </changeSet>
+
+    <changeSet author="svonworl" id="categories_new_feature">
+        <addColumn tableName="organization">
+            <column name="categorizer" type="boolean" defaultValue="false">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
+        <!-- the following addColumn+addNotNullConstraint combo is necessary because the JPA-generated db wants the column "collection" to be non-null, but complains if there is an SQL-defined default value. -->
```

Comment:
this seems odd, I suspect we've used default values before

Reply:
Oh, sorry, my comment is wrong.  The JPA-generated code wants the discriminator column "dtype" not to have an SQL-defined default, thus the above two-step process.